package/jailer.sh

```diff
@@ -79,12 +79,18 @@ cp -l /usr/bin/kustomize.sh /opt/jail/$NAME
 # Hard link ssh into the jail
 cp -l /usr/bin/ssh /opt/jail/$NAME/usr/bin
 
+# Hard link nc into the jail
```

Comment:
Do we need to do anything to allow users that are upgrading to Rancher v2.6.4 to be able to use this new functionality?

Reply:
There is no action on the user side for them to start using this new functionality. Of course, they need to set the proxy 
 variables properly when installing Rancher. 

This might be a case that I do not think will happen: users upgrade the Rancher server from a previous version with a non-functional rancher-provisioned downstream cluster. however, since this is kind of a new feature, deleting the broken one then creating a new one will be the solution for this case.

Reply:
OK. I will have to think about whether there is a more elegant way to handle this upgrade scenario.

Reply:
@jiaqiluo Now that I have thought about it, I think it is unlikely that a user will upgrade an air-gapped setup with clusters that are broken like this because this change is also need to successfully provision clusters. Is that correct?